sklearn/model_selection/_search.py

```diff
@@ -4,40 +4,44 @@
 """
 from __future__ import print_function
 
-# Author: Alexandre Gramfort <alexandre.gramfort@inria.fr>,
```

Comment:
Not intentionally. It must be the default autopep8 settings in Eclipse. Is there a document with the expected autopep8 settings anywhere?